debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -226,6 +226,13 @@ public String getAlias() {
                     " in case something else is processed this transform can log it as warning, error or stop the" +
                     " process");
 
+    static final Field SCHEMA_NAME_SUFFIX = Field.create("debezium.schema.name.suffix")
```

Comment:
See above, I don't think the suffix and this option add much value (unless I'm missing something), and we should rather put the aggregate type into schema names.

Reply:
We can certainly do that; the suffix was merely a suggestion in the issue from @jpechane and I think using the aggregate type in the schema name works all the same.  I'll remove this and change the behavior accordingly.

Reply:
Cool, thanks!